recipes/pvlib-python/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.3.2" %}
+
+package:
+  name: pvlib-python
+  version: {{ version }}
+
+source:
+  fn: pvlib-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/58/3d/8431a36635ad7a06c2fa00c53769074279364bea74ab87f99389b8a90a45/pvlib-{{ version }}.tar.gz
+  md5: 552fcf215911b5bb7b7a2e5ad6a9a6e6
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.8.2
+    - pandas >=0.13.1
+    - pytz
+    - six
+
+  run:
+    - python
+    - numpy >=1.8.2
+    - pandas >=0.13.1
+    - pytz
+    - six
+
+test:
+  imports:
+    - pvlib
```

Comment:
It's not that hard, take a look at https://github.com/conda-forge/bob.extension-feedstock/blob/master/recipe/meta.yaml#L28 for example.